Rust article

odarbelaeze opened the pull request.

Commit:
Create index.md

Commit:
Merge branch 'source' into rust-why-article

odarbelaeze committed:
Finish article draft

odarbelaeze committed:
Link github repo

dianareche commented on src/pages/rust-intro/index.md:
```diff
@@ -0,0 +1,274 @@
+---
+title: "Una introducción ligera a rust"
```
"Una ligera introducción a Rust"

pabloalcain committed:
Update index.md

1. Recomiendo *fuertemente* no usar auto-fills para este tipo de archivos, sino mantener la idea una oración <-> una línea. Es mucho más fácil hacer el version control luego
2. Primera persona del plural
3. Ejemplos de tiempo de ejecución con la compilación release y debug?
4. Explicar, talvez con una nota al pie y un link, cuál es el método con el cual se calcula pi. es decir, por qué "funciona"

Commit:
Merge pull request #25 from pabloalcain/patch-1

Update index.md

Commit:
Add pi estimation explanation
